src/mono/wasm/runtime/exports.ts

```diff
@@ -191,6 +193,11 @@ function initializeImportsAndExports(
     replacements.fetch = runtimeHelpers.fetch;
     replacements.readAsync = readAsync_like;
     replacements.requireOut = module.imports.require;
+    const originalUpdateGlobalBufferAndViews = replacements.updateGlobalBufferAndViews;
```

Comment:
where is replacements.updateGlobalBufferAndViews initialized?

Reply:
`src/mono/wasm/runtime/cjs/dotnet.cjs.lib.js`, that's inserted into the emscripten code